.github/pull_request_template.md

```diff
@@ -0,0 +1,39 @@
+### 🛠️ Fixes #Issue_Number
+
+<!-- Example: Fixes #31 -->
+
+### Related Issue/Addition to code
+- Issue goes here
+
+### 👨‍💻 What's being changed:
+
+<!-- List all the proposed changes in your PR. -->
+
+### Type of change
+
+<!-- Please delete options that are not relevant. -->
+
+<!-- Follow the below conventions to check the box -->
```

Comment:
In the check boxes add one more box to ensure that user has got the issue assigned before creating the PR

Reply:
Line 32 is similar: 

`I claimed any associated issue(s) and they are not someone else's`

Do you want me to change the wording of this?

Reply:
Nvm it is okay